### DISCUSSION TITLE
One progress but still having technical problem.

### DISCUSSION THREAD

**Discussion by JuninKorea** [opengisch/QField#2697] [Q&A]
It is always hard to figure out what I do not know.

I have my project file but cannot load all of the layers.
Let me share the image that I saw when I try to open my qfield project.
I hope there is someone who knows how to handle it
![Screenshot_20220405-170919](https://user-images.githubusercontent.com/102969856/161709677-c63ad78f-d060-4de7-8ed1-549cebf11bab.jpg)
.

**Comment by suricactus** [comment]
1) How did move your project to your device? Using QFieldCloud or a cable?
2) I guess you have uploaded/transfered only one directory of your project to your device, and forgot to do so for the files listed above. Looking at your screenshot, I would say these files were stored outside your project directory. Please note the relative path and `..` in front of it.

**Reply by JuninKorea** [reply]
Thanks for your interest and reply.

1. I was using cable.
2. I saved all of layers in the same folder. I'm attaching the screen shot below.
![11111](https://user-images.githubusercontent.com/102969856/161712917-096957d0-d45b-4d9b-8ec1-63a3e5ed9e13.png)


Thanks again!

**Reply by suricactus** [reply]
Well, seeing the project is called "cloud_cloud_cloud.qgs" makes me think there was something going on with it?

Also, if you try to open this project file in QGIS, you will see similar error, because the datasources of your layer are not configured correctly.

**Reply by suricactus** [reply]
@JuninKorea ping?
